configs/_base_/datasets/voc0712.py

```diff
@@ -1,6 +1,6 @@
 # dataset settings
 dataset_type = 'VOCDataset'
-data_root = 'data/VOCdevkit/'
+data_root = '/home/SharedDatasets/VOC/'
```

Comment:
Please do not modify the path of data. You can use softlink on your own machine.